src/stats/hexbin.jl

```diff
@@ -126,11 +120,7 @@ function plot!(hb::Hexbin{<:Tuple{<:AbstractVector{<:Point2}}})
             d1 = ((_x - nx)^2 + (yweight * (_y - ny))^2)
             d2 = ((_x - nxs)^2 + (yweight * (_y - nys))^2)
 
-            is_grid1 = d1 < d2
```

Comment:
This variable is useful I think, without that I wouldn't quite remember what the comparison was for

Reply:
Sorry, restored.